relay-kafka/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 //!
 //!     // build the client
 //!     let kafka_client = builder.build();
-//!     
```

Comment:
I didn't do this change on purpose, linter is forcing me. But It should be like this anyway, so might as well I suppose.